denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -53,8 +50,60 @@ tutorial_data:
       - "<&e>You can also get daily rewards!"
     title: "<&6>Use <&b>/warps"
     subtitle: "Go to the server warp: Marketplace"
-    message: "<&6>/warp &eto the server warp: Marketplace"
```

Comment:
This line has an unparsed color code `&e`.

Reply:
this was a removed line.